.travis.yml

```diff
@@ -4,24 +4,6 @@ node_js:
   - 8
   - 7
   - 6
-env:
-  - TZ=GMT
-  - TZ='Pacific/Kiritimati'
-  - TZ='Etc/GMT+12'
-matrix:
-  exclude:
-    - node_js: 8
-      env: TZ='Pacific/Kiritimati'
-    - node_js: 8
-      env: TZ='Etc/GMT+12'
-    - node_js: 7
-      env: TZ='Pacific/Kiritimati'
-    - node_js: 7
-      env: TZ='Etc/GMT+12'
-    - node_js: 6
-      env: TZ='Pacific/Kiritimati'
-    - node_js: 6
-      env: TZ='Etc/GMT+12'
```

Comment:
This removes testing on different versions of node.  Is this deliberate?

Reply:
The `node_js:` is still testing on 4 different versions of node. Instead of spawning 6 jobs and running the different time zones it is now spawning 4 and running. The grunt test task is setting the `node TZ` env variable before executing the tests. Now when you execute locally you will get tests run against the different time zones and see failures before committing. Instead of getting a test run against your current time zone. Which could possibly see a failure since the previous way was executing against multiple timezones only when ran in travis.